docs/_data/index.yml

```diff
@@ -0,0 +1,128 @@
+ReadTools:
+  - version: 1.0.0-22-g0f906c0-SNAPSHOT
```

Comment:
The documentation itself is for the 1.0.0, nothing has or will be changed since then until this will be accepted (at least from the user side)